src/index.js

```diff
@@ -1,4 +1,6 @@
-require('es6-promise').polyfill();  // Polyfill `Promise`.
+// Polyfill `Promise`.
+global.Promise = global.Promise || require('promise-polyfill');
```

Comment:
Roger, its not like this could ever really run in a node environment anyhow :gun: